src/react/components/pages/editorPage/canvas.tsx

```diff
@@ -325,13 +331,15 @@ export default class Canvas extends React.Component<ICanvasProps, ICanvasState>
 
     /**
      * Method called when deleting a region from the editor
-     * @param {string} id the id of the deleted region
+     * @param {string} id the id of the selected region
      * @param {boolean} multiselect boolean whether region was selected with multiselection
      * @returns {void}
      */
     private onRegionSelected = (id: string, multiselect: boolean) => {
+        const selectedRegions = this.getSelectedRegions();
+        // const selectedRegion = selectedRegions.find((region) => region.id === id);
```

Comment:
Seems like you were on the write track with this commented out code... if you select a new region do we want to set that as the template??